src/applications/caregivers/components/NeedHelpFooter/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const NeedHelpFooter = () => (
+  <footer className="need-help-footer row">
+    <h5>Need Help?</h5>
+    <hr />
+    <span>Need help filling out this form? Call our toll-free number: </span>
+
+    <section>
+      <a href="tel:18772228387">1-877-222-8387</a>
```

Comment:
This is a great example of what I'm talking about. I'd break this out into:
```
const CONSTANTS = {
  PHONE_NUMBER_TOLL_FREE = {
    HREF: 18772228387,
    PUBLIC: '1-877-222-8387'
  };
};

<a href="tel:CONSTANTS.PHONE_NUMBER_TOLL_FREE.HREF">
  CONSTANTS.PHONE_NUMBER_TOLL_FREE.PUBLIC
</a>
```

Reply:
Will do next time they change

Reply:
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself